Maddori.Apple/Maddori.Apple/Screen/Home/UIComponent/KeywordCollectionViewCell.swift

```diff
@@ -38,8 +44,8 @@ final class KeywordCollectionViewCell: BaseCollectionViewCell {
     override func render() {
         contentView.addSubview(keywordLabel)
         keywordLabel.snp.makeConstraints {
-            $0.edges.equalTo(safeAreaLayoutGuide)
-            $0.edges.equalToSuperview().inset(SizeLiteral.keywordLabelXInset)
+            $0.top.bottom.equalToSuperview()
+            $0.leading.trailing.equalToSuperview()
```

Comment:
p4;

```suggestion
            $0.edges.equalToSuperview()
```
로 합칠 수 있습니다 !

Reply:
아으... 이거 제가 진저 PR 에도 남겨뒀었는데 오히려 제가 안했네요 ㅎ ㅠ
[5b35f2d](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/36/commits/5b35f2d6e52eb680d445bf26124e2db3245e60d1)